Makefile

```diff
@@ -24,7 +24,7 @@ MOCKS_PACKAGE = github.com/insolar/insolar/testutils
 GOBUILD ?= go build -mod=vendor
 GOTEST ?= go test -mod=vendor
 
-FUNCTEST_COUNT ?= 1
+FUNCTEST_COUNT ?= 3
```

Comment:
I'm afraid you can't do it. This variable is used in `make functest`. Developers use it locally as well. Please re-define environment variable on CI if necessary.